src/EnergyPlus/WindowManager.cc

```diff
@@ -7990,10 +7937,10 @@ namespace Window {
             }
 
             //     Back diffuse-diffuse IR reflectance
-            BlindIRreflBack = G(2) * (1.0 - fEdge) + fEdge * ri;
+            // Real64 BlindIRreflBack = G(2) * (1.0 - fEdge) + fEdge * ri; // Blind back IR reflectance
 
             //     Back IR emissivity
-            p(15) = max(0.0001, 1.0 - p(13) - BlindIRreflBack);
+            // p(15) = max(0.0001, 1.0 - p(13) - BlindIRreflBack);
```

Comment:
[src/EnergyPlus/WindowManager.cc:7996]:(style),[unreadVariable],Variable 'p(15)' is assigned a value that is never used.

actually none of them are used